indico/web/util.py

```diff
@@ -30,6 +30,31 @@ def inject_js(js):
     g.injected_js.append(Markup(js))
 
 
+def jsonify_template(template, _render_func=render_template, **context):
```

Comment:
Usually together with ajaxDialog (the comment in ajaxDialog.js mentions this function)